new-site/src/components/Hero.scss

```diff
@@ -0,0 +1,150 @@
+@import "./utils.scss";
+
+.hero-container {
+  --koros-height: 145px;
+  --hero-height: 520px;
+  --hero-small-left: calc((100vw - 100%) / 2);
+  --hero-large-content-padding: var(--spacing-s);
+  --hero-large-content-left: calc((100vw + (2 * var(--hero-large-content-padding)) - var(--container-width-xl)) / 2);
+
+  background-color: var(--color-coat-of-arms);
+  box-sizing: border-box;
+  position: relative;
+  left: calc(var(--hero-small-left) * -1);
+  width: 100vw;
+
+  @include x-large-and-above-screen {
+    left: calc(var(--hero-large-content-left) * -1);
+    padding-left: var(--spacing-s);
```

Comment:
If both left and padding-left are omitted, the hero looks ok to me. Maybe Im missing something 🤔

Reply:
- [ ] Omit left and padding-left